lib/features/dataset/display.dart

```diff
@@ -141,16 +167,110 @@ class _DatasetDisplayState extends ConsumerState<DatasetDisplay> {
 
     updateVariablesProvider(ref);
 
+    Map<String, String> rolesOption = {
+      'IGNORE': ''' Ignore this dataset during analysis.
```

Comment:
Where else in Rattle do we use all uppercase button labels? Is there a reason? Also, please use the same button style as ActivtiyButton so all buttons look the same. These standout as different to the other buttons in the app.

Reply:
done